calculators/referendum_supporters_calculator.rb

```diff
@@ -110,6 +115,53 @@ def fetch
 
   private
 
+  def augment_lists_with_committees_that_raised_money(supporting_by_measure_name, opposing_by_measure_name)
+    committees_with_contributions = ActiveRecord::Base.connection.execute(<<-SQL)
+    SELECT
+      oakland_committees."Filer_ID",
+      oakland_committees."Filer_NamL",
+      oakland_committees."Ballot_Measure",
+      oakland_committees."Ballot_Measure_Election",
+      oakland_committees."Support_Or_Oppose"
+    FROM combined_contributions
+    INNER JOIN oakland_committees
```

Comment:
I'm not sure I follow -- we don't want to include all committees, only the ones on the spreadsheets that have some rows in combined_contributions. So even though we don't select any columns from combined_contributions, we do need both tables. Maybe a clearer way to write it would be to select `FROM oakland_committees` and then do the INNER JOIN to combined_contributions.

Reply:
Okay, I'll update. I was pretty tired last night when doing this originally 😅 .